taccsite_cms/static/site_cms/css/src/_imports/trumps/s-header.css

```diff
@@ -139,13 +139,8 @@ Styleguide Trumps.Scopes.Header
 
 /* Create a line between search bar and login */
 /* FAQ: The line should only exist if both elements are present */
-.s-header .s-search-bar ~ .s-portal-nav::before {
-  content: "";
-  width: 1px;
-  /* FAQ: Not `height`, cuz `.s-search-bar` (via `.form-inline`) is flexbox */
-  align-self: stretch;
-
-  background-color: rgb(255,255,255);
+.s-header .s-search-bar ~ .s-portal-nav {
+  border-left: 1px solid env(--header-minor-border-color);
```

Comment:
**What?** This is multiple changes:
1. simplify how border is drawn—use a real border instead of fake one*
2. fix default theme incorrect color `rgb(255,255,255)`
3. support theme-specific color via `env(--header-minor-border-color)`

_\* The minor refactor is technically out of scope._